tests/test_context_processors.py

```diff
@@ -83,3 +83,37 @@ def func(request):
     assert 200 == resp.status
     txt = yield from resp.text()
     assert 'foo: 1, bar: 2, path: /' == txt
+
+
+@asyncio.coroutine
+def test_context_not_tainted(test_client, loop):
+
+    global_context = {'version': 1}
+
+    @aiohttp_jinja2.template('tmpl.jinja2')
+    @asyncio.coroutine
+    def func(request):
+        return global_context
+
+    app = web.Application(loop=loop, middlewares=[
+            aiohttp_jinja2.context_processors_middleware])
```

Comment:
Please don't pass `context_processors_middleware` but call `setup()` with `context_processors` parameter.
It's the *official* way.

I'd like to support it in unit tests too.